packages/atomic/src/utils/accessibility-utils.tsx

```diff
@@ -17,7 +18,58 @@ export function AriaLiveRegion(regionName: string) {
 
   return (component: InitializableComponent, setterName: string) => {
     Object.defineProperty(component, setterName, {
-      set: (message) => dispatchMessage(message),
+      set: (message: string) => dispatchMessage(message),
     });
   };
 }
+
+export interface FocusTargetController {
+  setTarget(element: HTMLElement | undefined): void;
+  focusAfterSearch(): void;
+}
+
+export function FocusTarget() {
+  return (component: InitializableComponent, setterName: string) => {
+    const {componentWillLoad} = component;
+
+    component.componentWillLoad = function () {
```

Comment:
The decorator is called on the class itself, not an instance, so I used `componentWillLoad` to scope variables to each instance.